src/samaware/signals.py

```diff
@@ -1,10 +1,14 @@
 from django.dispatch import receiver
 from django.urls import resolve, reverse
 from django.utils.translation import gettext_lazy as _
+from pretalx.common.signals import EventPluginSignal
 from pretalx.orga.signals import nav_event
 
 import samaware
 
+speaker_html = EventPluginSignal()
+submission_html = EventPluginSignal()
```

Comment:
I'm not really familiar with the signal system so far. As far as I understand, this defines two completely new signals?

If so, would it make sense to add a prefix like`samaware` or even `samaware_talk_overview`? Or are they named this genericly on purpose?

Reply:
Outside of this thread, it has been explained to me that the signal names are local within SamAware. So I'm happy with them.